source/OutfitInfoDisplay.cpp

```diff
@@ -310,12 +343,46 @@ void OutfitInfoDisplay::UpdateAttributes(const Outfit &outfit)
 	attributesHeight = 20;
 
 	bool hasNormalAttributes = false;
+
+	// These attributes are regularly negative on outfits, so when positive,
+	// tag them with "added" and show them first. They conveniently
+	// don't use SCALE or BOOLEAN_ATTRIBUTES.
+	static const vector<string> EXPECTED_NEGATIVE = {
+		"outfit space", "weapon capacity", "engine capacity", "gun ports", "turret mounts"
+	};
+
+	for(const string& attr : EXPECTED_NEGATIVE)
+	{
+		double value = outfit.Get(attr);
+		if(value <= 0)
+			continue;
+
+		attributeLabels.emplace_back(attr + " added:");
+		attributeValues.emplace_back(Format::Number(value));
+		attributesHeight += 20;
+		hasNormalAttributes = true;
+	}
+
 	for(const pair<const char *, double> &it : outfit.Attributes())
 	{
-		static const set<string> SKIP = {
-			"outfit space", "weapon capacity", "engine capacity", "gun ports", "turret mounts"
-		};
-		if(SKIP.count(it.first))
+		if(count(EXPECTED_NEGATIVE.begin(), EXPECTED_NEGATIVE.end(), it.first))
+			continue;
+
+		// Only show positive values here, with some exceptions.
+		// Negative values are usually handled as a "requirement"
+		if(static_cast<string>(it.first) == "required crew")
+		{
+			// 'required crew' is inverted - positive values are requirements.
+			if(it.second > 0)
+				continue;
+
+			// A negative 'required crew' would be a benefit, so it is listed here.
+		}
+		else if(IsNotRequirement(it.first))
+		{
+			// This attribute is always listed here, though they may be negative.
+		}
+		else if(it.second < 0)
```

Comment:
```suggestion
		else if(it.second < 0 && !IsNotRequirement(it.first))
```
This is just better than having an empty if.